### ISSUE TITLE
[CSS Layout] Various suggestions

### ISSUE THREAD

**Issue by pamelafox** [gdisf/teaching-materials#61]
Porting over from site.
- The timing of when span is introduced is a bit awkward. Maybe change.
  - Consider adding a slide on CSS3 features (border-radius, etc).
- Add position:fixed after position:absolute.
- Add an exercise (or modify final exercise) where the student moves their CSS into an external file.
- Consider adding slide about "getting help" when learning HTML: jsfiddle, IRC.
- Consider adding slide about how to find designers (99designs, dribbble).
- Consider adding slide on prototyping tools (like MyBalsamiq or code generators like jQuery Mobile).

**Comment by trucnguyen** [comment]
Broken image on Box Model slide: http://www.teaching-materials.org/htmlcss-1day/css-boxmodel/slides.html#slide2

**Comment by pamelafox** [comment]
Most of this is deprecated, as it's about the old slide deck.

The last bullet seems like a decent idea:
- Consider adding slide on prototyping tools (like MyBalsamiq or code generators like jQuery Mobile).
I wrote an article on that for Khan Academy when I made the HTML/CSS curriculum there:
https://www.khanacademy.org/computing/computer-programming/html-css/css-layout-properties/a/planning-your-webpage
But it's more a feature request than a bug.
